src/img/close-white.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Compress SVG.

Reply:
@thisisdano this was marked resolved but is not compressed.

Reply:
oh huh

Reply:
ahhh.... I compressed `close-white-alt`... one sec